teuthology/task/ansible.py

```diff
@@ -263,7 +264,10 @@ def execute_playbook(self, _logfile=None):
     def _handle_failure(self, command, status):
         failures = None
         with open(self.failure_log.name, 'r') as log:
-            failures = yaml.safe_load(log)
+            try:
+                failures = yaml.safe_load(log)
+            except yaml.parser.ParserError:
+                log.exception("Failed to load failure log...")
```

Comment:
Agreed. The failure log will get moved into $archive_dir/ansible_failures.yaml but they might not know that.